common/reviews/api/streaming-image-volume-loader.api.md

```diff
@@ -426,6 +447,7 @@ enum Events {
     CAMERA_MODIFIED = 'CORNERSTONE_CAMERA_MODIFIED',
 
     CAMERA_RESET = 'CORNERSTONE_CAMERA_RESET',
+    DISPLAY_AREA_MODIFIED = 'CORNERSTONE_DISPLAY_AREA_MODIFIED',

```

Comment:
I don't think this should have it's own even - there should be a way to get properties changed, or this should trigger camera modified events.  @sedghi - any thoughts about the event?  Other properties don't have their own events.